src-tauri/src/logging.rs

```diff
@@ -51,7 +49,8 @@ pub fn setup_logging() -> Result<(), fern::InitError> {
     Ok(())
 }
 
-pub fn get_log_file() -> PathBuf {
+pub fn get_log_filepath() -> PathBuf {
```

Comment:
Not to be nitty, but I think `get_log_path` would be the best name :sweat_smile: (no need to change though)

Reply:
Spent almost 5 mins debating it lol. Chose wrong😅. Will sneak the change in the next PR.